tests/functional/features/application_restricted.feature

```diff
@@ -148,7 +148,7 @@ Feature: Unattended Access
     And I create a new app
     And I add the attribute with key of apim-app-flow-vars and a value of { "pds" : { "app-restricted": { "update": true } }}
     And I add the scope urn:nhsd:apim:app:level3:reasonable-adjustment-flag
-    And I wait for 100 milliseconds
+    And I wait for 200 milliseconds
```

Comment:
This was another suspected issue behind a failure. The test scenario adds a scope that should _not_ allow updates. But when an update was made after the scope was added, the PATCH was accepted/200. 

A similar change was made for a another scenario in this test suite.